docs/internals/contributing/triaging-tickets.txt

```diff
@@ -153,8 +150,9 @@ ideas here.
 Someday/Maybe
 -------------
 
-This stage isn't shown on the diagram. It's used sparingly to keep track of
-high-level ideas or long-term feature requests.
+This stage isn't shown on the diagram. It's an old way to keep track of
+high-level ideas or long-term feature requests, which are now handled by the
+`new feature idea tracker`_.
```

Comment:
Currently, we mostly use "Someday/Maybe" to track work that we do want to do but can't do now because we are waiting for a particular date (such as a final release to Python etc). Once that date is arrived, it becomes accepted but indicates to others that we can't merge something for this "yet"

I'm thinking either we leave this as it was or rewrite it :thinking: for it now to be called "old" is not strictly true as we're still using it

Reply:
Could we change the original text from "It's used sparingly to keep track of high-level ideas or long-term feature requests." to "It's used sparingly to keep track of long-term changes." We could add Sarah's example of a change being blocked by a final release to Python since that will be a consistent case too.

Reply:
I think this is a good idea 
Maybe:
```
This stage isn't shown on the diagram. It's used sparingly to keep track of
long-term changes.

These tickets are uncommon and overall less useful since they don't describe
currently actionable issues.
```
I deleted the last sentence of the second paragraph and changed "concrete" to "currently"